components/table/Table.tsx

```diff
@@ -983,6 +983,29 @@ export default class Table<T> extends React.Component<TableProps<T>, TableState<
       };
     }
 
+    const tableWithPagination = () => {
+      const { pagination } = this.state;
+      if (this.hasPagination()) {
+        if (pagination.position === 'top') {
+          return [
+            this.renderPagination(),
+            table,
+          ];
+        } else if (pagination.position === 'both') {
+          return [
+            this.renderPagination(),
+            table,
+            this.renderPagination(),
+          ];
+        } else {
+          return [
+            table,
+            this.renderPagination(),
+          ];
+        }
+      }
+      return table;
+    };
```

Comment:
Could you refactor it to less `if`s? Like:

```javascript
{this.renderPagination('top')}
{table}
{this.renderPagination('bottom')}
```

Reply:
good idea, thanks